Instructions on how to create a cronjob for renewing Lets Encrypt certificates (#2263)
travis-ci/docs-travis-ci-com: user/enterprise/platform-tips.md

$ replicatedctl app start
```

> Automate renewal with a cron job

Reviewer: Should this be a header of some sort?

$ replicatedctl app start
```

### Automate renewal of your Let's Encrypt SSL Certificate with a cron job

Create a file such as `/home/ubuntu/renew-certs.sh` with the following in it:

> ```

Reviewer 2: ```suggestion
```sh
```

#!/bin/bash

set -u

function usage() {
  echo
  echo "Usage: $(basename $0)"
  echo
  echo "Simple script to renew certs, should be used via a cron. Assumes that certs are already set up."
  echo
  echo "See https://docs.travis-ci.com/user/enterprise/platform-tips/#use-a-lets-encrypt-ssl-certificate for initial setup info."
  echo

  exit 1
}

if [[ $# -ne 0 ]]; then
  usage
fi

replicatedctl app stop
sudo certbot renew
replicatedctl app start

> ```

Reviewer 2: ```suggestion
```{: data-file="./home/ubuntu/renew-certs.sh"}
```

Set the executable permissions you'll need by doing `chmod +x /home/ubuntu/renew-certs.sh`.

> You'll then want to create a cronjob by using your favorite text editor, for example:

Reviewer 2: nano is like nobodies favourite editor! :-p How about 

> Edit `/etc/crontab` by appending the following lines:

```
nano /etc/crontab
```

> Then append the below to the file that you've just opened in your text editor. Make sure to adjust the configuration with values that apply to the certificate you are creating.

Reviewer 2: How about adding a specific example instead the file comment.

```
# Renews certs at 2am on the 1st of February, May, August, and November.
# Please change the configuration that applies to the certificate you are creating.
0 2 1 2,5,8,11 * /home/ubuntu/renew-certs.sh
```

> **Note: Be aware that this process will also introduce downtime.**

Reviewer 2: You can use quote `>` to add note formatting.

```suggestion
> This process will introduce a small amount of downtime while the certificates are renewed.
```

> Make sure to plan to communicate with your users every time this downtime will happen so they are aware that their builds will temporarily be stopped until the certificate gets renewed and your Travis CI Enterprise installation comes back online.

Reviewer 2: ```suggestion
Make sure to communicate with your users before each renewal so they are aware that their builds will temporarily be stopped until the certificate gets renewed.
```
